backends/advanced/src/advanced_omi_backend/controllers/websocket_controller.py

```
@@ -302,22 +302,22 @@ async def _initialize_streaming_session(
-        if parakeet_url:
-            provider = "parakeet"
-        elif deepgram_key:
-            provider = "deepgram"
-        else:
-            raise ValueError("No transcription provider configured (DEEPGRAM_API_KEY or PARAKEET_ASR_URL required)")
+    # Determine transcription provider from config.yml
+    from advanced_omi_backend.model_registry import get_models_registry
+
+    registry = get_models_registry()
+    if not registry:
+        raise ValueError("config.yml not found - cannot determine transcription provider")
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)